src/fswAlgorithms/transDetermination/ephemNavConverter/ephemNavConverterAlgorithm.cpp

```diff
@@ -0,0 +1,47 @@
+/*
+ ISC License
+
+ Copyright (c) 2025, Laboratory for Atmospheric Space Physics, University of Colorado at Boulder
+
+ Permission to use, copy, modify, and/or distribute this software for any
+ purpose with or without fee is hereby granted, provided that the above
+ copyright notice and this permission notice appear in all copies.
+
+ THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+ WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+ MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+ ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+ WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+ ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+ OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+ */
+
+#include "fswAlgorithms/transDetermination/ephemNavConverter/ephemNavConverterAlgorithm.h"
+#include "architecture/utilities/linearAlgebra.h"
+
+/*! Reset method for the ephemNavConverter algorithm.
+ @return void
+ @param callTime [ns] Time the method is called
+ */
+void EphemNavConverterAlgorithm::reset(uint64_t callTime) {
```

Comment:
:large_orange_diamond: For the algorithm let's remove the reset function when empty. For the module it makes sense to explicitly implement the basilisk module interface.